src/app/games/awale/awale.component.ts

```diff
@@ -66,12 +61,13 @@ export class AwaleComponent extends RectangularGameComponent<AwaleRules,
         ];
         this.filledCoords = [];
     }
-    private showPreviousMove(): void {
-        const previousMove: AwaleMove = this.rules.node.move.get();
+    public override showLastMove(move: AwaleMove): void {
+        const lastPlayer: number = this.getState().getCurrentPlayer().value;
+        this.last = MGPOptional.of(new Coord(move.x, lastPlayer));
         const previousState: AwaleState = this.rules.node.mother.get().gameState;
```

Comment:
si tu te sens de les remplacer par this.getPreviousState() qui est déjà défini

Reply:
NOICE ! DONE MULTERU